src/intl/ka/page-index.json

```diff
@@ -62,7 +62,7 @@
   "page-index-contribution-banner-image-alt": "ლეგოთი შექმნილი ეთერეუმის ლოგო.",
   "page-index-contribution-banner-button": "მეტი კონტრიბუციის შესახებ",
   "page-index-tout-upgrades-title": "ცოდნის გაუმჯობესება",
-  "page-index-tout-upgrades-description": "ეთერეუმი შედგება ურთიერთდაკავშირებული განახლებებისაგან, რომლებიც შექმნილია იმისათვის, რომ ქსელი უფრო მასშტაბური, უსაფრთხო და მდგრადი გახდეს.",
+  "page-index-tout-upgrades-description": "125.",
```

Comment:
Not sure what happened here... 🤔 

```suggestion
  "page-index-tout-upgrades-description": "ეთერეუმი შედგება ურთიერთდაკავშირებული განახლებებისაგან, რომლებიც შექმნილია იმისათვის, რომ ქსელი უფრო მასშტაბური, უსაფრთხო და მდგრადი გახდეს.",
```

<img width="1370" alt="Screenshot 2022-09-26 at 11 13 40" src="https://user-images.githubusercontent.com/62268199/192251530-2c1b4a7b-86c7-4ed5-bccb-031d5b92f32f.png">

<img width="1676" alt="Screenshot 2022-09-26 at 11 14 16" src="https://user-images.githubusercontent.com/62268199/192251666-3902ad21-b2b8-44eb-a40d-1fdbcbabd738.png">